kuma/core/middleware.py

```diff
@@ -14,6 +15,11 @@
 from .views import handler403
 
 
+SITE_HOST = urlparse(settings.SITE_URL).netloc
```

Comment:
I'm not sure if this approach is a good one or not. I thought it was nice that it was "automatic", in the sense that no new environment variables needed to be added, and it captures (I think) all of the current use cases (local development, testing, SCL3, AWS), but on the other hand, adding an environment variable, something like `CDN_HOST` would make this more configurable. @jwhitlock I would specifically like your opinion on this.